app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListAdapter.kt

```diff
@@ -123,16 +123,19 @@ class ThreadListAdapter(
         }
     }
 
-    private fun getDisplayedRecipient(thread: Thread): Recipient? = thread.messages.lastOrNull()?.from?.firstOrNull()
+    private fun Thread.getDisplayedMessage() = messages.lastOrNull()
```

Comment:
Is it possible to have threads that have no messages?

Reply:
No. I've updated it.